src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DefaultProjectSnapshotManager.cs

```diff
@@ -193,7 +193,7 @@ public override void DocumentAdded(HostProject hostProject, HostDocument documen
             {
                 var loader = textLoader is null
                     ? DocumentState.EmptyLoader
-                    : (() => textLoader.LoadTextAndVersionAsync(Workspace, documentId: null, CancellationToken.None));
+                    : (() => textLoader.LoadTextAndVersionAsync(new LoadTextOptions(), CancellationToken.None));
```

Comment:
When I tried to not give it a hash algo it caused test failures. Was LoadTextOptions updated to default to something? If so what?

Reply:
I'm not sure - this appeared to pass everything except integration tests - I was under the impression it didn't matter because you override the implementation of `LoadTextAndVersionAsync` and that override ignores the inputs.

Reply:
As discussed offline, I mis-spoke here, the errors in question were (hidden) runtime errors, not test failures.